### PR TITLE
Edits for /ok endpoint

### LINKED ISSUE #754: [FEATURE] Add GET /ok endpoint to App Config Building Block
**Is your feature request related to a problem? Please describe.**
Currently, we do not have a health check endpoint that returns HTTP 200. We need this for better health checks using AWS.

**Describe the solution you'd like**
1. Please add a GET /ok endpoint that returns HTTP 200 with JSON ```{"success": "true"}```.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by minump**
## Description
Adding /ok public endpoint for healthcheck. This public endpoint returns a success message with HTTP 200 response.

**Fixes #754 **

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [x] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.11.1] - 2021-08-11
 ### Changed
 - Temporarily turn off group authentication on image get endpoints.[#750](https://github.com/rokwire/rokwire-building-blocks-api/issues/750)
+### Added
```
Comment (sandeep-ps) [inline CHANGELOG.md:14]: Please move this under the Unreleased section. Thanks.

**Comment by sandeep-ps** [comment]
Hi Minu, Please update the CHANGELOG. The rest looks good. Thanks.

**Comment by minump** [comment]
committed the change. Should I create another pull request for the same?

**Comment by sandeep-ps** [comment]
> committed the change. Should I create another pull request for the same?

You can update your local branch 754-add-ok- ... and push your changes to the remote branch. It will update the PR.

**Comment by minump** [comment]
> > committed the change. Should I create another pull request for the same?
> 
> You can update your local branch 754-add-ok- ... and push your changes to the remote branch. It will update the PR.

i changed in remote branch. i also see that the PR now has 2 commits. i guess the PR is updated. let me know if I am missing anything.

**Comment by sandeep-ps** [comment]
> > > committed the change. Should I create another pull request for the same?
> > 
> > 
> > You can update your local branch 754-add-ok- ... and push your changes to the remote branch. It will update the PR.
> 
> i changed in remote branch. i also see that the PR now has 2 commits. i guess the PR is updated. let me know if I am missing anything.

Yes, that's right: https://github.com/rokwire/rokwire-building-blocks-api/pull/761/files

I will merge this after reviewing the other two PRs in this repo. Thanks. You can start with your next task from a new branch from develop branch. Thanks.

**Review by sandeep-ps — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,7 +16,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Fix get event endpoint with user auth key. [#746](https://github.com/rokwire/rokwire-building-blocks-api/issues/746)
+
 ### Added
+- Fix published capabilities and talents not showing without login. [#767](https://github.com/rokwire/rokwire-building-blocks-api/issues/767)
```
Comment (sandeep-ps) [inline CHANGELOG.md:21]: It looks like when you resolved merge conflicts, new sections got added. We need to update the existing entries under the [Unreleased] section. Please let me know if you have any questions. Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**